charts/k8s-monitoring/docs/examples/log-metrics/README.md

```diff
@@ -0,0 +1,59 @@
+<!--
+(NOTE: Do not edit README.md directly. It is a generated file!)
+(      To make changes, please modify values.yaml or description.txt and run `make examples`)
+-->
+# Log Metrics
+
+This example shows how to generate metrics based on the log data. It utilizes a `stage.metric` processor within the
+[loki.process](https://grafana.com/docs/alloy/latest/reference/components/loki/loki.process/#stagemetrics) component
+to generate metrics captured from the Pod logs. This is a simple metric, but other metrics can be added, or more complex
+stages can be utilized to filter and match on certain log lines and build metrics from them.
+
+## Values
+
+```yaml
+---
+cluster:
+  name: log-metrics-example
+
+destinations:
+  - name: prometheus
+    type: prometheus
+    url: http://prometheus.prometheus.svc:9090/api/v1/write
+
+  - name: loki
+    type: loki
+    url: http://loki.loki.svc:3100/loki/api/v1/push
+
+podLogs:
+  enabled: true
+
+  extraLogProcessingStages: |-
```

Comment:
In https://github.com/grafana/k8s-monitoring-helm/blob/25c2df7b95c35f902242b2e9314856b99420b99a/charts/k8s-monitoring/charts/feature-node-logs/templates/_module.alloy.tpl#L150 there is no pod filter. Does that mean if we have multiple stages, each stage gets applied to every single pod log being scraped? Isn't it ineffective?